libpod/healthcheck.go

```diff
@@ -131,10 +131,14 @@ func (c *Container) runHealthCheck() (HealthCheckStatus, error) {
 	logrus.Debugf("executing health check command %s for %s", strings.Join(newCommand, " "), c.ID())
 	timeStart := time.Now()
 	hcResult := HealthCheckSuccess
-	hcErr := c.Exec(false, false, []string{}, newCommand, "", "", streams, 0)
+	ec, hcErr := c.Exec(false, false, []string{}, newCommand, "", "", streams, 0, nil, "")
```

Comment:
It might be time to make the arguments to exec a struct, this is getting out of hand

Reply:
I'll leave this open but I think it'd be better to add that as a follow up